padinfo/padinfo.py

```diff
@@ -482,11 +489,12 @@ async def otherinfo(self, ctx, *, query: str):
 
         alt_monsters = PicViewState.get_alt_monsters_and_evos(dgcog, monster)
         discrep = dgcog.database.graph.monster_is_discrepant(monster)
+        qsettings = QuerySettings.extract(await self.get_fm_flags(ctx.author), query)
```

Comment:
btw i havent said anything but fm_flags is kinda confuse, would like to name this to get_query_flags or get_id3_flags or get_flags_from_find_monster or something, we can figure out a better name later